source/includes/cli-reference/_projectfile.html.md

```diff
@@ -53,6 +53,13 @@ There must be at least one valid path in the `files` list.
 points to a pre-configured simulation container inside the platform. The
 `command` field describes the command to run to start the simulator.
 
+Current list of supported simulators for Docker cloud-hosted training:
+
+ * [`openai.gym`][1]
+ * [`bonsai.python`][2]
+ * [`bonsai.energyplus`][3]
+ * `bonsai.gazebo`

```

Comment:
gazebo won't be in the release tomorrow (2017.05.31 release being deployed on Tuesday June 5th). We may want to remove it from here and add it when gazebo is ready.